jdr/src/main/java/org/jboss/as/jdr/util/JdrZipFile.java

```diff
@@ -151,7 +152,7 @@ public void addAsString(InputStream stream, String path) throws Exception {
 
     public String getProductDirName() {
         if(this.productDirName == null)
-            this.productDirName = String.format("%s-%s", this.env.getProductName().replace(" ", "_").toLowerCase(), this.env.getProductVersion().split("\\.")[0]);
+            this.productDirName = String.format("%s-%s", this.env.getProductName().replace(" ", "_").toLowerCase(Locale.getDefault()), this.env.getProductVersion().split("\\.")[0]);
```

Comment:
For WF and EAP, Locale.ENGLISH is more correct as we're not going to use anything else for this.

But WF is extensible, so it's possible community users are using something else, so +1 to this. It's the same behavior as what we currently have.

Reply:
Why is this the only place that calls (redundantly, as Brian noted) `Locale.getDefault()`?

Reply:
I think explicitly saying 'Locale.getDefault()' is preferable to saying nothing, as it let's us distinguish an intentional choice from the common case of just forgetting to specify a locale.